native/database.cc

```diff
@@ -74,7 +75,7 @@ void write_patient_to_buffer(uint32_t start_unique,
                              std::vector<uint32_t>& buffer) {
     if (current_patient.birth_date < epoch) {
         throw std::runtime_error(
-            absl::StrCat("Cannot have a birth date before epoch (1900) ",
+            absl::StrCat("Cannot have a birth date before epoch (1800) ",
```

Comment:
Can we add some comments here on why these values are chosen? A new person walking in will have zero idea why 1800 was chosen as a sensible minimum birthdate.